core-services/08-sysctl.sh

```diff
@@ -2,15 +2,19 @@
 
 if [ -x /sbin/sysctl -o -x /bin/sysctl ]; then
     msg "Loading sysctl(8) settings..."
+    mkdir /run/sysctl
     for i in /run/sysctl.d/*.conf \
         /etc/sysctl.d/*.conf \
         /usr/local/lib/sysctl.d/*.conf \
         /usr/lib/sysctl.d/*.conf \
         /etc/sysctl.conf; do
 
-        if [ -e "$i" ]; then
+        fname="/run/sysctl/$(basename $i)"
```

Comment:
I think it would be better to use substitutions instead of executing `/usr/bin/basename` here.

`fname=${i##*/}`

Reply:
Done, thank you.